kfs-core/src/main/java/edu/arizona/kfs/sys/batch/service/impl/RoomImportServiceImpl.java

```diff
@@ -23,151 +23,153 @@
-        PrintWriter outReportErrorWriter = null;
-        PrintWriter outReportWriter = null;
-        String routeCodeValue = "";
-        Map<String, String> routecodeToCampuscodeMap;
-        CampusBo campus;
-        
-        try {
-        	outReportWriter = setupReportOutputFiles(outReportWriter);
-        	outReportErrorWriter = setupErrorReportOutputFiles(outReportErrorWriter);
-            writetoReportOutputFilesHeader(outReportWriter, outReportErrorWriter);
-            Collection<ArchibusRooms> archibusRooms = businessObjectService.findAll(ArchibusRooms.class);
-            for (ArchibusRooms archibusRoom : archibusRooms) {
-                LOG.debug("Processing Record: " + archibusRoom.toString());
-                reportMsg = "";
-                matchingRoom = new Room();
-                matchingBuilding = new Building();
-
-
-                // look up building Alpha (buildingCode) by building ID(BuildingId)
-                Map<String, String> fieldValues = new HashMap<String, String>();
-                fieldValues.put("buildingId", archibusRoom.getBuildingCode());
-                Collection<ArchibusBuildings> archibusBuildings = (Collection<ArchibusBuildings>) businessObjectService.findMatching(ArchibusBuildings.class, fieldValues);
-                
-                if(archibusBuildings.size() < 1){
-                    LOG.debug("There is no building for BL_ID of " + archibusRoom.getBuildingCode());
-                    reportMsg += "There is no building for BL_ID of " + archibusRoom.getBuildingCode() + ".  Room not added: " + archibusRoom.getBuildingRoomNumber();
-                    writetoErrorReportOutputFile(outReportErrorWriter, reportMsg);
-                    continue;
-                }
-                
-                for (ArchibusBuildings archbuilding : archibusBuildings) {
-                    buildingCode = archbuilding.getBuildingCode();
-                    routeCodeValue = archbuilding.getRouteCode();
-                    break;
-                }
-
-                // Look up campusCode by using routecode value
-                routecodeToCampuscodeMap = buildingAndRoomImportDao.PopulateRoutecodeToCampusCodeMap();
-                if (StringUtils.isBlank(routecodeToCampuscodeMap.get(routeCodeValue))) {
-                    campusCode = KFSParameterKeyConstants.MAIN_CAMPUSCODE;
-                }
-                else {
-                    campusCode = routecodeToCampuscodeMap.get(routeCodeValue);
-
-                    // Look for a valid Campus Code in KFS
-                    Map<String, String> pkeys = new HashMap<String, String>();
-                    pkeys.put("campusCode", campusCode);
-                    pkeys.put("active", "Y");
-                    campus = (CampusBo) businessObjectService.findByPrimaryKey(CampusBo.class, pkeys);
-
-                    if (ObjectUtils.isNull(campus)) {
-                        LOG.debug("Campus Code is not valid, record not saved to building");
```

Comment:
Same as above.  Adding debug/info messages will help if there is a batch failure.